library/src/environments/environment.ts

```diff
@@ -4,7 +4,8 @@
 
 export const environment = {
   production: false,
-  apiUrl: 'https://bank.demo.cybrid.app'
+  sandboxBankApiBasePath: 'https://bank.demo.cybrid.app',
+  productionBankApiBasePath: 'https://bank.production.cybrid.app'
 };
 
 /*
```

Comment:
We should have four environments:

1. Demo- the current Heroku environment
2. Staging- the AWS Staging environment
3. Sandbox- the AWS Sandbox environment
4. Production- the AWS Production environment